index.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+entries:
+  boldreports:
+  - apiVersion: v2
+    appVersion: 5.3.10
```

Comment:
Give current release version for app version and version of helm chart.

Reply:
Changed into current release version